src/main/java/com/appland/appmap/process/hooks/ToggleRecord.java

```diff
@@ -177,7 +201,7 @@ public static void doFilterJakarta(Event event, Object[] args) throws ExitEarly
         junit = true;
       }
     }
-    IRecordingSession.Metadata metadata = new IRecordingSession.Metadata();
+    RecordingSession.Metadata metadata = new RecordingSession.Metadata();
```

Comment:
There's only one type of RecordingSession now - it streams events to a temp file.